src/app/Home.tsx

```diff
@@ -133,12 +131,10 @@ export default function Home(props: Props) {
                 />
               }
               label={
-                'Teaching ' +
-                (typeof props.latestSemester !== 'undefined' &&
-                props.latestSemester.message === 'success'
-                  ? 'in ' +
-                    displaySemesterName(props.latestSemester.data, false)
-                  : 'Next Semester')
+                props.latestSemester == ''
+                  ? 'Teaching Next Semester'
+                  : 'Teaching in' +
```

Comment:
Can you add a space here?
<img width="322" height="58" alt="image" src="https://github.com/user-attachments/assets/f9301873-0e3f-4ba5-ab80-7be87bb63774" />